lib/ledger-node-operations.js

```diff
@@ -7,6 +7,35 @@
 
 const async = require('async');
 const validator = require('./validator');
+const bedrock = require('bedrock');
+const {config: {constants}} = bedrock;
+const validate = require('bedrock-validation').validate;
+
+const validateContext = (context) => {
+  if(!context) {
+    throw new Error('Operation must include a context.');
+  }
+  if(!(typeof context === 'string' || Array.isArray(context))) {
+    throw new Error('Operation context must be a string or an array.');
+  }
+  if(typeof context === 'string'
+      && context !== constants.WEB_LEDGER_CONTEXT_V1_URL) {
+    throw new Error('Operation does not contain the WebLedgerContext.' +
+                    `'@context': ${context}`);
+  }
+  if(Array.isArray(context)
+     && context !== constants.WEB_LEDGER_CONTEXT_V1_URL[0]) {
+    throw new Error('Operation must contain WebLedgerContext as the' +
+                    `first entry. '@context': ${context}`);
+  }
```

Comment:
The above needs a number of style fixes (indentation, move `&&` up, etc).
`WebLedgerContext` should use the URL instead of that text.
You should use BedrockErrors and pass the details (the context) that way.
The messages are missing spaces where they break lines.